### DISCUSSION TITLE
Can't deploy Java web app to Azure

### DISCUSSION THREAD

**Discussion by pennywise-01** [community/community#29602] [Actions]
Can anyone help me regarding this issue, I want to set an action to deploy to Azure. The build job is okay but the deploy job show one error and i could not figure that out. This is the error:
![image](https://user-images.githubusercontent.com/54774679/184780998-da789647-1c54-4557-9722-2210f4f76126.png)

this is the configuration:
![image](https://user-images.githubusercontent.com/54774679/184781056-9d798b82-0964-4076-ab6a-c9894157a094.png)

Incase you need the whole link of my repo, here is the link https://github.com/Hafzan-250601/bookstore

**Comment by chikeb-aymen** [comment]
you solved the problem??

**Reply by pennywise-01** [reply]
Still cant solve this problem
